code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -417,7 +415,324 @@
 				/obj/structure/constructshell = 50 //Fuck it we ball what could go wrong
 				)
 
+/obj/effect/spawner/lootdrop/trade_sol/federation_minor
+	name = "11. trans-solar federation small gear"
+	loot = list(
+				/obj/item/storage/box/enforcer_rubber = 50,
+				/obj/item/storage/box/buck = 30, //Only seven shots, make them count
+				/obj/item/lighter/zippo/gonzofist = 50,
+				/obj/item/clothing/glasses/welding/superior = 50,
+				/obj/item/clothing/suit/armor/bulletproof = 50,
+				/obj/item/clothing/mask/gas/explorer/marines = 50,
+				/obj/item/clothing/gloves/combat = 50,
+				/obj/item/storage/belt/military/assault = 50,
+				/obj/item/clothing/under/solgov = 50,
+				/obj/item/flag/solgov = 30
+				)
+
+/obj/effect/spawner/lootdrop/trade_sol/federation_major
+	name = "12. trans-solar federation large gear"
+	lootcount = 3
+	loot = list(
+				/obj/item/storage/box/deagle = 20, //One mag
+				/obj/item/gun/projectile/automatic/pistol/m1911 = 30, //Again, one mag. Don't lose it.
+				/obj/item/melee/baseball_bat/homerun = 50,
+				/obj/item/rcd/combat = 50,
+				/obj/item/weaponcrafting/gunkit/universal_gun_kit/sol_gov = 50,
+				/obj/item/mod/module/dispenser = 50,
+				/obj/item/mod/module/noslip = 40,
+				/obj/item/clothing/mask/holo_cigar = 50
+				)
+
+/obj/effect/spawner/lootdrop/trade_sol/cybersun_minor
+	name = "11. cybersun industries small gear"
+	loot = list(
+				/obj/item/storage/box/syndidonkpockets = 50,
+				/obj/item/clothing/suit/jacket/syndicatebomber = 50,
+				/obj/item/storage/box/syndie_kit/space = 50,
+				/obj/item/clothing/glasses/meson/cyber = 50,
+				/obj/item/storage/pill_bottle/zoom = 50,
+				/obj/item/clothing/mask/gas/voice_modulator/chameleon = 50,
+				/obj/item/mecha_parts/mecha_equipment/weapon/energy/xray = 30,
+				/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/carbine = 30,
+				/obj/item/flag/syndi = 30
```

Comment:
Should uhh... Should they be openly selling syndicate flags?

Reply:
Don't some literally sell a dark marauder?

Reply:
Cybersun is being "changed" to Robomoon, as a legally distinct and totally-not-related-to-the-syndicate shell company. In this case, the flags are a surplus - I could make them chameleon flags though?